Bobsar0/Fast-Food-Fast
# #161461655 Consume create menu endpoint

#### What does this PR do?
Enables admin to add food item to menu

#### Description of Task to be completed?
- Implement food image upload/processing using multer
- Add script to create food menu item
- Create admin UI form to 'add food item to menu'
- Add script to get all menu items from db upon click of menu link

#### How should this be manually tested?
- Clone the repo and run the following commands:
```
  - git checkout ft-consume-create-menu-endpoint-#161461655 
  - npm install
  - npm start
```
- Navigate to http://localhost:9999/ and login/create an account using the credentials below
``` 
 username: admin
 email: admin@fastfoodfast.com
 password: admin!23
```
_You should be redirected to a dedicated menu page_
- Navigate to `http://localhost:9999/templates/admin.html`
- Click on **Administrator Panel** > **Add food item to menu**
- Fill in the form that appears with details of food item you wish to add, upload a picture and save
- Click on the menu link, your food item should be displayed in the specified genre section

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/#161461655
